clients/fides-js/src/lib/tcf/events.ts

```diff
@@ -7,6 +7,10 @@ import { FIDES_SEPARATOR } from "./constants";
  * `vendors_disclosed` and our own AC string addition.
  */
 export const fidesEventToTcString = (event: FidesEvent) => {
+  if (!window.Fides.options.fidesTcfGdprApplies) {
+    // A `null` TC string means gdpr does not apply
+    return null;
+  }
```

Comment:
Three nitpicky suggestions:
* rename to `extractTCStringForCmpApi`, to be clear that this is used specifically for the CmpApi
* add a return type of `string | null` for type checks 👍 
* add a comment to link to the IAB docs

Putting those together (no guarantee this compiles!):
```suggestion
* `vendors_disclosed` and our own AC string addition.
 *
 * Returns a string to be used for the `CmpApi` class from `@iabtechlabtcf/cmpapi`, which expects one of three input strings:
 *  - Encoded TC string (e.g. "CP1sGZVP1..."), which will be decoded into `TCData` and set `gdprApplies = true`
 *  - Empty string (e.g. `""`), which will populate an empty `TCData`, but importantly set `gdprApplies = true`
 *  - Null string (e.g. `null`), which will also populate an empty `TCData`, but set `gdprApplies = false`
 * 
 * Therefore we should only use a `null` string when we specifically want to set `gdprApplies = false`!
 *
 * See the `CmpApi` docs for more details: https://github.com/InteractiveAdvertisingBureau/iabtcf-es/blob/master/modules/cmpapi/README.md#trigger-change-event
 */
export const extractTCStringForCmpApi = (event: FidesEvent): string | null => {
  if (!window.Fides.options.fidesTcfGdprApplies) {
    // A `null` TC string will instruct CmpApi to set gdprApplies = false
    return null;
  }
```